tests/README.md

```diff
@@ -26,15 +42,8 @@ devices. For example:
 JSON requires commas between list items, but make sure to omit the comma after
 the last list item.
 
-Then, run `cargo test` as root. (Root permissions are needed to access block
-devices, and to create device-mapper targets.)
-
-## Logging
-
-Integration test logging is done via the crate env_logger. See
-[here](https://doc.rust-lang.org/log/env_logger/) for details.
-
-Example: To enable logging for specific modules set:
+Then:
 
-`RUST_LOG=blockdev_tests=debug,pool_tests=debug,util::test_results=debug`
+> make test-real
 
+Each tests is run in a separate process.
```

Comment:
Fix grammar, or does this sentence really need to be stated, and can be removed?